src/subscriptions/subscriptions.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { Controller, Request, Body, Post, HttpCode } from '@nestjs/common';
+import { SubscriptionsService } from './subscriptions.service';
+
+@Controller('subscriptions')
+export class SubscriptionsController {
+  constructor(private readonly subscriptionsService: SubscriptionsService) {}
+
+  @Post('create')
+  async create(
+    @Request() req,
+    @Body()
+    {
+      accountId,
+      subscription,
+    }: { accountId: string; subscription: object },
+  ): Promise<void> {
+    try {
+      await this.subscriptionsService.createSubscription(
+        accountId,
+        JSON.stringify(subscription),
+      );
+      console.log(`Subscription for account ${accountId} has been saved.`);
+    } catch (e: any) {
+      console.error('Subscription creation failed.', e);
+      // TODO map error
```

Comment:
is console.log sufficient to enable logs to show up in GCP?

Reply:
Yes, this should be sufficient (similarly to Pagoda Console).